lib/util/exec/index.ts

```diff
@@ -87,6 +87,11 @@ function getRawExecOptions(opts: ExecOptions): RawExecOptions {
 
   // Set default max buffer size to 10MB
   rawExecOptions.maxBuffer = rawExecOptions.maxBuffer ?? 10 * 1024 * 1024;
+
+  if (opts.ignoreStdout) {
```

Comment:
please add a test inside exec utils so we've full coverage there without needing gradle tests